### ISSUE TITLE
11.0.0 TER version does not work on 11LTS while 9.2.0 TER version does

### ISSUE THREAD

**Issue by vmans** [CodersCare/device#1]
Like the title says, in my case, device version 9.2.0 does work on the latest 11LTS and device version 11.0.0 does not work on the latest 11LTS.

Difference is obviously the matomo detection in 11 versus the piwik detection in 9.

I ran this compare:

![image](https://user-images.githubusercontent.com/25581914/179059546-7cb772d2-9245-4dd6-8162-41688e15c179.png)

Based on that I added:

`use TYPO3\CMS\Core\Utility\ExtensionManagementUtility;
`

and

```
        if (!class_exists('Spyc')) {
            require_once ExtensionManagementUtility::extPath('device') . '/Contrib/mustangostang/spyc/Spyc.php';
        }
        if (!class_exists('DeviceDetector\DeviceDetector')) {
            require_once ExtensionManagementUtility::extPath('device') . '/Contrib/matomo/device-detector/autoload.php';
        }
```

to UserAgentConditionProvider.php

and now 11.0.0 works in 11LTS.

Perhaps this is not what you had in mind, but it works for me for now.

**Comment by Cybercraft-GmbH** [comment]
Sorry - that was our fault, because there needs to be a specific TER version that does not just provide a contrib folder but requires those classes explicitely, which is not necessary for the composer based version. So we just missed to put exactly those lines into the TER version.

Will be fixed with the next release.

**Comment by Cybercraft-GmbH** [comment]
Fixed with 9.3.1, 10.0.2 and 11.0.1 - in 9.x and 10.x the path was still wrong, while in 11.x it was missing completely.
